reimbursement.md

```diff
@@ -271,13 +257,13 @@ Reimbursement `status` is set to `pending` while waiting for the card scheme to
 
 ### Status
 
-Find the reimbursement status in the transaction `reimbursement.status` property:
+Find the reimbursement status in the reimbursement's `status` property:
```

Comment:
Small one, but I'd say we can refactor this, now that it's much simpler to find.

```suggestion
The `status` property represents the possible reimbursement status as:
```